index.js

```diff
@@ -2179,6 +2184,13 @@
     };
   }
 
+  //  see :: (String, { name :: String, url :: String }) -> String
+  function see(label, record) {
+    return record.url &&
```

Comment:
I think very few library authors do. I've considered this matter and reached the conclusion that it may be better that if your dependency exposes anything to the user, it should be a peer dependency rather than a regular dependency. And if your library works with input from another library, you should specify its version range using an [optional dependency](https://docs.npmjs.com/files/package.json#optionaldependencies). But I'm still reflecting upon these thoughts.